editor/project_converter_3_to_4.cpp

```diff
@@ -2241,24 +2241,25 @@ void ProjectConverter3To4::process_gdscript_line(String &line, const RegExContai
 
 	// rotating = true  ->   ignore_rotation = false # reversed "rotating" for Camera2D
 	if (contains_function_call(line, "rotating")) {
-		int start = line.find("rotating");
+		String function_name = "rotating";
+		int start = line.find(function_name);
 		bool foundNextEqual = false;
-		String line_to_check = line.substr(start + String("rotating").length());
+		String line_to_check = line.substr(start + function_name.length());
 		String assigned_value;
 		for (int current_index = 0; line_to_check.length() > current_index; current_index++) {
 			char32_t chr = line_to_check.get(current_index);
 			if (chr == '\t' || chr == ' ') {
 				continue;
 			} else if (chr == '=') {
 				foundNextEqual = true;
-				assigned_value = line.right(current_index).strip_edges();
+				assigned_value = line.substr(function_name.length() + start + current_index + 1).strip_edges();
```

Comment:
This seems more natural to me as it matches the actual positions in the string (first the starting point, then move by the function name length, then add the index of whitespace to skip).

```suggestion
				assigned_value = line.substr(start + function_name.length() + current_index + 1).strip_edges();
```

I haven't tested but are you sure the `+ 1` is correct? Or does it just happen to work because you're converting a string with a space after `=`? This code is meant to handle both `rotating = true` and `rotating=true` and any other combination of whitespace characters (which then get removed by `strip_edges`).

Reply:
The `+ 1` is needed to keep the `=` character out of the substring Without the `+ 1`, the resulting substring would be `=true` (or `= true`, depending on whether there's a space or not). I have tested multiple all of the following cases on the same project and they all converted correctly.
```
rotating = true
rotating=true
rotating =true
rotating= true
rotating  =  true
rotating     = true
rotating    =      true
rotating   =    true
rotating = false
rotating=false
rotating =false
rotating= false
rotating  =  false
rotating     = false
rotating    =      false
rotating   =    false
```

Reply:
Indeed, thanks for confirming!